tests/jerry/es.next/regression-test-issue-4341.js

```diff
@@ -0,0 +1,57 @@
+// Copyright JS Foundation and other contributors, http://js.foundation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+function run_test(test_str, expected) {
+  // print(test_str, " != ", expected);
+  assert(eval(test_str) === expected);
+}
+
+var test_minus_one = [
+  "-1n >> (2n ** 32n)",
+  "-1n >> 2n",
+  "-1n >> (2n ** 1n)",
+  "-5n >> 3n",
+  "-5n >> 4n",
+  "-(2n ** 32n) >> (2n ** 31n)",
+  "-(2n ** 32n) >> (2n ** 32n)",
+  "-(2n ** 32n) >> (2n ** 33n)",
+  "-(2n ** 32n) >> (2n ** 33000n)",
+  "-(2n ** 32n) >> 32n",
+
+  /* (xn << -yn) ==> (xn >> yn) */
+  "-1n << -(2n ** 32n)",
+  "-1n << -2n",
+  "-1n << -(2n ** 1n)",
+  "-5n << -3n",
+  "-5n << -4n",
+  "-(2n ** 32n) << -(2n ** 31n)",
+  "-(2n ** 32n) << -(2n ** 32n)",
+  "-(2n ** 32n) << -(2n ** 33n)",
+  "-(2n ** 32n) << -(2n ** 33000n)",
+
+];
+
+for (var idx = 0; idx < test_minus_one.length; idx++) {
+  run_test(test_minus_one[idx], -1n);
```

Comment:
What is the advantage of this loop? It feels harder to find an error from the backtrace compared to call `run_test` for each item. I don't think we need to optimize this code at all.

Reply:
Wanted to avoid typing/copying the `run_test(.., -1n);` string :) . But I'll remove the loop.